### DISCUSSION TITLE
[Bug]: Accessing a file whitch other than index.php

### DISCUSSION THREAD

**Discussion by yrpn** [beyondcode/herd-community#1020] [Q&A]
### Platform

Windows

### Operating system version

Windows 11 (23H2)

### System architecture

Intel (x86)

### Herd Version

1.11.0

### Bug description

In a laravel project when a put a php file to the public directory, I can't reach it from browser.
For example when I put the phpinfo.php file with `<?php phpinfo()` content and I try to reach it from example.test/phpinfo.php
In this case I'll be redirected back to example.test and got a 404 site

**Comment by mpociot** [comment]
Laravel's routing does not work this way.
You should take a look at Laravel's routing documentation: https://laravel.com/docs/11.x/routing#basic-routing

**Reply by yrpn** [reply]
I know the laravel's routing, but this is nginx config issue, because on shared host it works
